README.md

```diff
@@ -104,6 +105,35 @@ const baz = "bar" + bar
 />
 ```
 
+## lineNumbers
+
+When `lineNumbers` is enabled, react-refractor will append line number data just like the [Prism's Line Numbers plugin](https://prismjs.com/plugins/line-numbers/). However you should provide your own styling. E.g.:
```

Comment:
This seems to work well for highlighting that doesn't wrap - soft-wrapping requires a bit of client-side trickery and measuring, so I'm OK with not including that logic for now - but we should make a note of it here.

Reply:
I've added one at the end of this section.